bs/src/tablecloth.mli

```diff
@@ -1527,41 +1527,345 @@ module Result : sig
+    root(0.0) |> and_then ~f:recip = None
+    ```
+  *)
   val and_then : f:('a -> 'b option) -> 'a option -> 'b option
 
+  (**
+    `or_ opt_a opt_b` (`or_(opt_a, opt_b)` in ReasonML) returns
+    `opt_a` if it is of the form `Some x` (`Some(x) in ReasonML);
+    otherwise, it returns `opt_b`.
+    
+    ### Example
+    
+    ```ocaml
+    or_ (Some 11) (Some 22) = Some 11
+    or_ None (Some 22) = Some 22
+    or_ (Some 11) None = Some 11
+    or_ None None = None
+    ```
+
+    ```reason
+    or_(Some(11), Some(22)) == Some(11);
+    or_(None, Some(22)) == Some(22);
+    or_((Some(11)), None) == Some(11);
+    or_(None, None) == None;
+    ```
+  *)
   val or_ : 'a option -> 'a option -> 'a option
 
+  (**
+    `orElse(opt_a, opt_b)` returns `opt_b` if it is of the form `Some(x)`;
+    otherwise, it returns `opt_a`.
+    
+    (Same as `or_else`.)
+    
+    ### Example
+    
+    ```reason
+    orElse(Some(11), Some(22)) == Some(22);
+    orElse(None, Some(22)) == Some(22);
+    orElse((Some(11)), None) == Some(11);
+    orElse(None, None) == None;
+    ```
+
+  *)
   val orElse : 'a option -> 'a option -> 'a option
 
+  (**
+    `or_else opt_a opt_b` returns `opt_b` if it is of the form `Some x`;
+    otherwise, it returns `opt_a`.
+    
+    (Same as `orElse`.)
+    
+    ### Example
+    
+    ```ocaml
+    orElse (Some 11) (Some 22) = Some 22
+    orElse None (Some 22) = Some 22
+    orElse (Some 11) None = Some 11
+    orElse None None = None
+    ```
+  *)
   val or_else : 'a option -> 'a option -> 'a option
 
+  (**
+    `map ~f:fcn opt` (`map(~f = fcn, opt)` in ReasonML) returns
+    `fcn x` (`fcn(x)` in ReasonML) if `opt` is of the form
+    `Some x` (`Some(x)` in ReasonML); otherwise, it returns `None`.
+    
+    ### Example
+    
+    ```ocaml
+    map ~f:(fun x -> x * x) (Some 9) = Some 81
+    map ~f:(fun x -> x * x) None = None
+    ```
+    
+    ```reason
+    map(~f = (x) => x * x, Some(9)) == Some(81)
+    map(~f = (x) => x * x, None) == None
+    ```
+  *)
   val map : f:('a -> 'b) -> 'a option -> 'b option
 
+  (**
+    `withDefault(~default = defVal, opt)` If `opt` is of the form `Some(x)`,
+    this function returns `x`. Otherwise, it returns the default value `defVal`.
+    
+    (Same as `with_default`.)
+    
+    ### Example
+    
+    ```reason
+    withDefault(~default = 99, Some(42)) == 42;
+    withDefault(~default = 99, None) == 99;
+    ```
+  *)
   val withDefault : default:'a -> 'a option -> 'a
 
+  (**
+    `with_default(~default: def_val, opt)` If `opt` is of the form `Some x`,
+    this function returns `x`. Otherwise, it returns the default value `def_val`.
+    
+    (Same as `withDefault`.)
+    
+    ### Example
+    
+    ```ocaml
+    with_default ~default:99 (Some 42) = 42
+    with_default ~default:99 None = 99
+    ```
+  *)
   val with_default : default:'a -> 'a option -> 'a
 
+  (**
+    `foldrValues(opt, xs)` When `opt` is of the form `Some(x)`,
```

Comment:
We're removing this so no doc required.

Reply:
Removed doc, but not the definition, so that the build can succeed.